tests/test_websocket_handler.c

```diff
@@ -487,14 +489,15 @@ static int s_tester_init(struct tester *tester, struct aws_allocator *alloc) {
     struct aws_websocket_handler_options ws_options = {
         .allocator = alloc,
         .channel = tester->testing_channel.channel,
-        .initial_window_size = SIZE_MAX,
+        .initial_window_size = s_default_initial_window_size,
         .user_data = tester,
         .on_incoming_frame_begin = s_on_incoming_frame_begin,
         .on_incoming_frame_payload = s_on_incoming_frame_payload,
         .on_incoming_frame_complete = s_on_incoming_frame_complete,
         .manual_window_update = s_tester_options.manual_window_update,
     };
     tester->websocket = aws_websocket_handler_new(&ws_options);
+    testing_channel_drain_queued_tasks(&tester->testing_channel);
     ASSERT_NOT_NULL(tester->websocket);
```

Comment:
trivial: could you move the `drain_queued_tasks()` after the `ASSERT_NOT_NULL(tester->websocket);`?

It's weird to put an arbitrary function between calling a function and asserting its results